src/Microsoft.SqlTools.Authentication/Microsoft.SqlTools.Authentication.csproj

```diff
@@ -9,6 +9,7 @@
 		<PackageReference Include="Microsoft.Data.SqlClient" />
 		<PackageReference Include="Microsoft.Identity.Client" />
 		<PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" />
+		<PackageReference Include="Newtonsoft.Json" />
```

Comment:
couldn't see what this was being added for?

Reply:
Might have left this out after experiments, will clean up !